infra/base-images/base-runner/coverage

```diff
@@ -139,6 +153,13 @@ done
 # Wait for background processes to finish.
 wait
 
+if [[ $FUZZING_LANGUAGE == "go" ]]; then
+  /root/go/bin/gocovmerge $DUMPS_DIR/*.profdata > fuzz.cov
+  go tool cover -html=fuzz.cov -o $REPORT_ROOT_DIR/index.html
+  cp $REPORT_ROOT_DIR/index.html $REPORT_PLATFORM_DIR/index.html
+  echo "End for go targets"
+else
+
```

Comment:
please add one indentation level for all the commands from here down to line 190

Reply:
ok, I had not done this at first for easier review ;-)

Reply:
smart!